src/core/server/http/router/router.ts

```diff
@@ -267,9 +268,9 @@ export class Router implements IRouter {
   }
 
   private async handle<
-    P extends ObjectType,
-    Q extends ObjectType,
-    B extends ObjectType | Type<Buffer> | Type<Stream>
+    P extends ObjectType | ValidateFunction<unknown>,
+    Q extends ObjectType | ValidateFunction<unknown>,
+    B extends ObjectType | Type<Buffer> | Type<Stream> | ValidateFunction<unknown>
```

Comment:
given the number of usages in our code, I think type aliases for 
- `ObjectType | ValidateFunction<unknown>`
- `ObjectType | Type<Buffer> | Type<Stream> | ValidateFunction<unknown>`

should be a good idea. Will also avoid heavy diffs in further validation type updates.